docs/install/containers/recipes/index.md

```diff
@@ -118,3 +118,43 @@ RUN <<EOT
 
 After making these changes, build the project container as usual.
 It will no longer output the access log, but will continue to output the event log.
+
+
+## Pack the ZODB
+
+A common maintenance task of a Plone instance is to [pack the ZODB](https://zodb.org/en/stable/reference/zodb.html#ZODB.DB.pack).
+Packing removes old revisions of objects.
+It is similar to [routine vacuuming in PostgreSQL](https://www.postgresql.org/docs/8.3/routine-vacuuming.html).
+
+The official {doc}`/install/containers/images/backend` container and project containers based on them have a `pack` command to pack the ZODB.
+The command will work in standalone mode, ZEO mode, and RelStorage mode but only with PostgreSQL.
```

Comment:
This sentence is confusing. "but only with PostgreSQL" is meant to be a qualifier only when using it with Relstorage, but it sounds like it applies to all the options, which makes no sense (it's only _possible_ to use postgres if you're doing via Relstorage).